roles/openshift_cluster_metrics_hawkular/tasks/main.yaml

```diff
@@ -0,0 +1,56 @@
+---
+
+- name: Copy Configuration to temporary conf
+  command: >
+    cp {{ openshift.common.config_base }}/master/admin.kubeconfig {{hawkular_tmp_conf}}
+  changed_when: false
+
+- name: Install cluster metrics templates
+  copy:
+    src: cluster-metrics
+    dest: "{{ openshift.common.config_base }}/"
+
+- name: Create metrics-deployer Service Account
```

Comment:
I'll take a look as soon as I can.

Outside of the scope of this, but I was thinking earlier about making the service account role delegate to the first master and then we could just depend on it where necessary.